### PR TITLE
fix : use event identifier in session link

### LINKED ISSUE #5581: Email: New Session Proposal does not provide correct link
The proposal email does not provide correct link, e.g. it is: https://eventyay.com/events/598/sessions/5201

screenshot from 2019-01-23 17-43-00

### PR DESCRIPTION
**Pull request by uds5501**
Fixes #5581 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
The current session link format uses `event_id` instead of `identifier` as it should be used.

#### Changes proposed in this pull request:

- using `identifier` instead of `id` in the link

### CODE REVIEW

**Comment by uds5501** [comment]
@iamareebjamal @CosmicCoder96 Please review

File: app/api/sessions.py [hunk 1]
```diff
@@ -51,8 +52,9 @@ def after_create_object(self, session, data, view_kwargs):
             organizer = session.event.get_organizer()
             organizer_email = organizer.email
             frontend_url = get_settings()['frontend_url']
+            concerned_event = EventModel.query.filter(Event.id == session.event_id).first()
```
Comment (iamareebjamal) [inline app/api/sessions.py]: No need, you already have session.event

Comment (uds5501) [reply app/api/sessions.py]: @iamareebjamal Understood, will use it in next commit

File: app/api/sessions.py [hunk 2]
```diff
@@ -51,8 +51,9 @@ def after_create_object(self, session, data, view_kwargs):
             organizer = session.event.get_organizer()
             organizer_email = organizer.email
             frontend_url = get_settings()['frontend_url']
+            concerned_event = session.event
```
Comment (CosmicCoder96) [inline app/api/sessions.py]: keep the name as `event` for this variable. We are doing the same for the organizer.

Comment (uds5501) [reply app/api/sessions.py]: done

**Comment by uds5501** [comment]
@iamareebjamal @CosmicCoder96 Please review :)

**Pull request merged**